tasks/size_contracts.js

```diff
@@ -120,25 +129,39 @@ task(
     {
       content: chalk.bold(`Change (${config.unit})`),
     },
+    {
+      content: chalk.bold(`Init Size (${config.unit})`),
+    },
+    {
+      content: chalk.bold(`Init Change (${config.unit})`),
+    },
   ]);
 
   let oversizedContracts = 0;
 
   for (let item of outputData) {
-    if (!item.size) {
+    if (!item.size || !item.initSize) {
```

Comment:
Is it possible that a contract could have `size` of 0 and `initSize` of 999999999?  Such as a contract that consists of only an excessively large constructor.  In that case, I think we should only check `initSize` here.

Reply:
When exactly can `size` exceed `initSize`?  I suppose that could happen if the initcode does something like `extcodecopy` and uses that as the deployed bytecode?

This may have been a null check at some point, but I see that those values are always set, so the `=== 0` comparison you added is good.

Reply:
I'm not sure if it can? Maybe with some trickery but I'm not sure. My thinking here is that this script relies on artifacts with separate "bytecode" and "deployedBytecode" fields, and it is at least possible to create that json file with one field present and the other empty. Happy to change if you'd prefer.